02-rendering-logic-ii/src/components/ControlPane/ControlPane.module.css

```diff
@@ -14,8 +12,7 @@
 
 .title {
   margin: 0;
-  /* Optical centering */
-  margin-bottom: -4px;
+  line-height: 1;
```

Comment:
Looks way more reliable than using the "optical centering trick" (which depends on `px`), as it is just a single line anyway

Reply:
Actually, this is only good if it's a single line with a short text, because it will look bad if wrapped.